apps/wing/src/cli.ts

```diff
@@ -151,6 +152,11 @@ async function main() {
         .choices(["tf-aws", "tf-azure", "tf-gcp", "sim", "awscdk"])
         .default("sim")
     )
+    .addOption(
+      new Option("-l, --log <LogType>", "Log type")
+        .choices([LogType.DEFAULT, LogType.NONE, LogType.EXTENDED])
+        .default(LogType.DEFAULT)
+    )
```

Comment:
How do these logging levels line up with the log levels in the Wing Console? And what's the difference between `--log=extended` and `--debug` CLI options?

I feel like some more discussion is needed about logging. Maybe we can leave this as a follow up enhancement since it wasn't part of the original issue

<img width="171" alt="Screenshot 2023-07-31 at 4 04 01 PM" src="https://github.com/winglang/wing/assets/5008987/3ae0b9a1-c284-4173-888d-3075c56c02af">

Reply:
I think you're right, maybe it will be good for this PR to allow `--debug` to behave like `-log=extended`.
I'll open an issue for thinking of a better and more systematic approach

Reply:
+1 that this all needs to align with the console

Reply:
https://github.com/winglang/wing/issues/3680